src/marqo/tensor_search/tensor_search.py

```diff
@@ -2648,3 +2691,33 @@ def delete_documents(config: Config, index_name: str, doc_ids: List[str]):
             document_ids=doc_ids,
         )
     )
+
+def normalize_vector(embeddings: Union[List[List[float]], ndarray]) -> List[List[float]]:
```

Comment:
I'd recommend moving this out of the `tensor_search.py` file. We're trying to improve our repo structure and reduce the size of the files. Maybe it could be under `core/utils` or `core/inference`? What do you think @papa99do

Reply:
we can keep it here for now. I'll move it out in my refactoring PR after this is merged.